native/src/overlay/menu.rs

```diff
@@ -289,13 +289,15 @@ where
         use std::f32;
 
         let limits = limits.width(Length::Fill).height(Length::Shrink);
-        let text_size = self.text_size.unwrap_or(renderer.default_size());
 
         let size = {
+            #[allow(clippy::or_fun_call)]
             let intrinsic = Size::new(
                 0.0,
-                f32::from(text_size + self.padding.vertical())
-                    * self.options.len() as f32,
+                f32::from(
+                    self.text_size.unwrap_or(renderer.default_size())
```

Comment:
`self.text_size.unwrap_or_else(|| renderer.default_size())` better.
A little memory optimization here and not have to make an exclusion for the linter.

I would even make a separate structure with Default and Deref into u16. Then you could do `.unwrap_or_default()`.

Especially since such an expression is very often used in the code.

Reply:
This also returns a u16 which is stack allocated. Since it will be allocated in the stack we don't need lazy eval, and it's very cheap anyways (ideally `default_size` would be marked as `const`, but trait const fn's aren't finished yet), so we don't need the indirection.

Reply:
On second thought; Rust should figure out and just inline the value, bypassing the lazyness.